src/activities/UseFmeService.ts

```diff
@@ -16,23 +16,214 @@ export interface UseFmeServiceInputs {
+        | "security/categories/{category}/resources/{resource}/roles/{role}/permissions/{permission}"
+        | "security/categories/{category}/roles"
+        | "security/categories/{category}/roles/{role}"
+        | "security/categories/{category}/roles/{role}/permissions/{permission}"
+        | "security/ldap/servers"
+        | "security/ldap/servers/{server}"
+        | "security/ldap/servers/{server}/importrole"
+        | "security/ldap/servers/{server}/importuser"
+        | "security/ldap/servers/{server}/role"
+        | "security/ldap/servers/{server}/roles"
+        | "security/ldap/servers/{server}/synchronize"
+        | "security/ldap/servers/{server}/user"
+        | "security/ldap/servers/{server}/users"
+        | "security/roles"
+        | "security/roles/{role}"
+        | "security/roles/{role}/accounts"
+        | "security/roles/{role}/accounts/{account}"
+        | "security/roles/{role}/ldap"
+        | "systemevents/configurations"
+        | "systemevents/configurations/{configuration}"
+        | "systemevents/configurations/{configuration}/enabled"
+        | "systemevents/events/completed"
+        | "systemevents/schema/events"
+        | "systemevents/schema/events/{configuration}"
+        | "systemevents/schema/parameters"
+        | "systemevents/schema/parameters/{configuration}"
+        | "tokens"
+        | "tokens/{user}/{name}"
+        | "tokens/{user}/{name}/categories"
+        | "tokens/{user}/{name}/categories/{category}"
+        | "tokens/{user}/{name}/enabled"
+        | "tokens/{user}/{name}/resources"
+        | "tokens/{user}/{name}/resources/{category}/{resource}"
+        | "transformations/engines"
+        | "transformations/jobroutes/defaulttag"
+        | "transformations/jobroutes/tags"
+        | "transformations/jobroutes/tags/{tag}"
+        | "transformations/jobroutes/tags/{tag}/engines"
+        | "transformations/jobroutes/tags/{tag}/engines/{engine}"
+        | "transformations/jobroutes/tags/{tag}/repositories"
+        | "transformations/jobroutes/tags/{tag}/repositories/{repository}"
+        | "transformations/jobs/active"
+        | "transformations/jobs/active/{jobid}"
+        | "transformations/jobs/completed"
+        | "transformations/jobs/completed/{jobid}"
+        | "transformations/jobs/id/{jobid}"
+        | "transformations/jobs/id/{jobid}/log"
+        | "transformations/jobs/id/{jobid}/request"
+        | "transformations/jobs/id/{jobid}/result"
+        | "transformations/jobs/queued"
+        | "transformations/jobs/queued/{jobid}"
+        | "transformations/jobs/running"
+        | "transformations/jobs/running/{jobid}"
+        | "transformations/submit/{repository}/{workspace}"
+        | "transformations/transact/{repository}/{workspace}"
+        | "transformations/transactdata/{repository}/{workspace}"
+        | "user/favorites/workspace"
+        | "user/favorites/workspace/{repository}/{workspace}"
```

Comment:
This should be `user/favorites/workspaces/{repository}/{workspace}`. (plural)

Reply:
Fixed